common/src/com/thoughtworks/go/domain/builder/Builder.java

```diff
@@ -101,23 +94,23 @@ public int hashCode() {
     }
 
     public void cancel(DefaultGoPublisher publisher, EnvironmentVariableContext environmentVariableContext, TaskExtension taskExtension) {
-        publisher.consumeLineWithPrefix("Start to execute cancel task: " + cancelBuilder.getDescription());
+        publisher.consumeLineWithPrefix("Cancel task: " + cancelBuilder.getDescription());
```

Comment:
I think `Cancel task <task1>` would incorrectly imply that task1 was being cancelled. Users can setup and `on-cancel-task` for a given job which gets executed when that job execution has been cancelled. The description being logged here was for the on-cancel task not the original task that was being executed.